src/styles/blocks/card/blockscard/card.scss

```diff
@@ -0,0 +1,45 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid $border-card;
+  border-radius: 5px;
+  transition: 300ms linear;
+  cursor: pointer;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0;
+    color: $main-text-color;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $second-text-color;
+    text-align: left;
+    margin-left: 0;
+    margin-top: 4px;
+    margin-bottom: 0;
+  }
+}
+
+.card:hover {
+  transform: scale(1.2);
```

Comment:
The transform property can cause layout shifts, which can lead to performance issues. Instead of scaling the entire card, consider changing the properties that have less impact on performance, such as color or box-shadow.